hamilton/plugins/h_opentelemetry.py

```diff
@@ -0,0 +1,130 @@
+import json
+import logging
+from contextvars import ContextVar
+from typing import Any, Collection, List, Optional, Tuple
+
+logger = logging.getLogger(__name__)
+
+
+try:
+    from opentelemetry import context, trace
+    from opentelemetry.sdk.trace import Span
+except ImportError as e:
+    raise ImportError("Failed to import `opentelemetry`") from e
```

Comment:
We should add instructions with the right installable target.

Reply:
https://github.com/DAGWorks-Inc/hamilton/blob/f35f6b7fd4abdeabacae49c354b6943925b89dc0/hamilton/plugins/h_ddog.py#L11 for example